### PR TITLE
Add Data Tracker's Corridor Retiming view to config

### PR DESCRIPTION
**Pull request by chiaberry**
https://github.com/cityofaustin/atd-data-tech/issues/11181

Knack side, Prefect PR TBD

### CODE REVIEW

File: services/config/knack.py [hunk 1]
```diff
@@ -169,6 +169,12 @@
             "location_field_id": "field_182",
             "socrata_resource_id": "h4cy-hpgs",
         },
+        "view_3814": {
+            "description": "Corridor retiming",
+            "scene": "scene_1526",
+            "modified_date_field": "field_4169",
+            "socrata_resource_id": "bp5z-kciq",
+        }
```
Comment (chiaberry) [inline services/config/knack.py:177]: everything else is fixing typos in docs

**Review by Charlie-Henry — APPROVED** [review]
🚢 🚢 🚢 

I have Knack services already ported to Prefect 2 in [my feature branch](https://github.com/cityofaustin/atd-prefect/blob/ch-prefect2-atd-knack-services/flows/knack/atd_knack_markings_contractor_work_orders.py). Not a problem if you want me to quickly just add another flow for this ETL.

File: services/utils/shared.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 def format_keys(record):
     """Format Knack record keys by converting to lower case and replacing space
     with underscores"""
-    return {key.lower().replace(" ", "_"): val for key, val in record.items()}
+    return {key.lower().replace(" ", "_").replace("-", "_"): val for key, val in record.items()}
```
Comment (chiaberry) [inline services/utils/shared.py]: @johnclary & @Charlie-Henry, the column that is named re-timing status is named re_timing_status in socrata, but the script before this change was converting it into re-timing_status. I should probably check other datasets to see if this is going to break something, right?

Comment (johnclary) [reply services/utils/shared.py]: i have no concerns! this looks good.

Comment (johnclary) [reply services/utils/shared.py]: @chiaberry the README references this behavior in two places (search for `underscore`). would you update that?

**Pull request merged**